doc/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -275,6 +275,26 @@ files that you will need to start using MoveIt!
 * Congratulations!! - You are now done generating the configuration
   files you need for MoveIt!
 
+
+Gazebo Simulation
+------------------------------------
+
+You can use the generated robot urdf file in the simulation folder to
+spawn the robot in Gazebo.
+
+* Use rosrun to start gazebo empty world: ::
+
+   roslaunch gazebo_ros empty_world.launch paused:=true use_sim_time:=false gui:=true throttled:=false recording:=false debug:=true
+
+* Use rosrun to spawn the robot: ::
```

Comment:
shouldn't this be a roslaunch file instead?

Reply:
I think it is a `rosrun` at least according to the [`roslaunch` tutorial](http://gazebosim.org/tutorials?tut=ros_roslaunch) .